website/client/components/notifications.vue

```diff
@@ -210,7 +210,11 @@ export default {
   },
   watch: {
     baileyShouldShow () {
-      this.$root.$emit('show::modal', 'new-stuff');
+      if (this.user.flags.newStuff) {
+        this.$root.$emit('show::modal', 'new-stuff');
+      } else {
+        this.$root.$emit('hide::modal', 'new-stuff');
```

Comment:
does this get triggered every time the page loads?

Reply:
this only gets triggered on the flag change. On initial loading, the show/hiding of the modal is handled by `checkUserAchievements()`